topostats/io.py

```diff
@@ -172,6 +174,110 @@ def save_folder_grainstats(
             LOGGER.info(f"No folder-wise statistics for directory {_dir}, no grains detected in any images.")
 
 
+def read_null_terminated_string(open_file: io.TextIOWrapper) -> str:
+    """Read an open file from the current position in the open binary file,
+    until the next null value.
+
+    Parameters
+    ----------
+    open_file: io.TextIOWrapper
+        An open file object.
+
+    Returns
+    -------
+    str
+        String of the ASCII decoded bytes before the next null byte.
+    """
+    byte = open_file.read(1).decode("ascii")
+    value = ""
+    while byte != "\x00":
+        value += byte
+        byte = open_file.read(1).decode("ascii")
+    return str(value)
+
+
+def read_u32i(open_file: io.TextIOWrapper) -> str:
```

Comment:
I was unsure about this myself too. My reasoning for putting it in the base `io.py` is that the functions aren't inherent to loading scans, and are more of general I/O than loading scans. Why would we want to force a LoadScans instance to be created when the functions could just be standalone?

**However** I will gladly defer to your judgement on this if you think they should go in LoadScans.

Reply:
Given this some thought and they probably shouldn't be methods of LoadScans, but then they don't warrant a class of their own either.

One option to keeping the code organised though would be to group them into a sub-module such as `topostats/io/binary.py` (`binary` because I can't think of a better name!). Would require adjusting the existing imports, moving tests and importing them to `topostats/io.py` though.

If you think this is worthwhile I'd say make it a separate issue and merge this then rejig things around, otherwise this PR gets dragged out.